tests/manage/z_cluster/nodes/test_nodes_maintenance.py

```diff
@@ -49,6 +53,16 @@ def init_sanity(self):
         """
         self.sanity_helpers = Sanity()
 
+    @pytest.fixture(autouse=True)
```

Comment:
Please use the fixture from https://github.com/red-hat-storage/ocs-ci/blob/1d55e2b6e8f052773837f3de7102c9a1a1d2c602/tests/conftest.py#L853

Reply:
Please ignore. the mentioned fixture will take place only for specific tiers.